src/plugins/condition/StyleRuleManager.js

```diff
@@ -27,22 +27,26 @@ export default class StyleRuleManager extends EventEmitter {
         super();
         this.openmct = openmct;
         this.callback = callback;
-        this.refreshData = this.refreshData.bind(this);
-        this.toggleSubscription = this.toggleSubscription.bind(this);
-        if (suppressSubscriptionOnEdit) {
-            this.openmct.editor.on('isEditing', this.toggleSubscription);
-            this.isEditing = this.openmct.editor.editing;
-        }
 
         if (styleConfiguration) {
             this.initialize(styleConfiguration);
             if (styleConfiguration.conditionSetIdentifier) {
-                this.openmct.time.on("bounds", this.refreshData);
+                this.openmct.time.on("bounds", (bounds, isTick) => {
```

Comment:
using arrow functions in here in lieu of bind:
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

Reply:
I was also surprised

Reply:
This is going to create a memory leak because it's not being removed in `destroy()`. Being an anonymous function makes it hard to remove as well, which I think is why this was a bound, named function. I also agree that binding should work here, and we should figure out why it is apparently not.

Reply:
Ok! I’ll take a look